src/rgw/rgw_common.h

```diff
@@ -508,6 +508,33 @@ struct RGWAccessKey {
 };
 WRITE_CLASS_ENCODER(RGWAccessKey)
 
+struct MGWAccessKey {
```

Comment:
can we use RGWAccessKey struct itself in sal, or any motr specific struct required can be created in sal layer, let's not add in rgw_common

Reply:
>can we use RGWAccessKey struct itself in sal

`RGWAccessKey` has id, key and subuser attributes but we need id, key and user_id, thats why I created a new structure. 

> any motr specific struct required can be created in sal layer, let's not add in rgw_common

Agree. I have moved it to `rgw_sal_motr.h`